### PR TITLE
Cert Management: Adding OCP v4 templates for use with OpenShift Applier

### PR DESCRIPTION
**Pull request by oybed**
#### What is this PR About?
Adding support for cert-manager / OCPv4 for use with openshift-applier

#### How do we test this?
Tweak the params, then run the openshift-applier with the inventory

cc: @redhat-cop/day-in-the-life

### CODE REVIEW

File: cert-manager-configs/.applier/group_vars/seed-hosts.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+
+openshift_domain: "my.cluster.example.com"
+cert_manager_namespace: "cert-manager"
+
+cert_manager_params:
+  DNS_DOMAIN: "{{ openshift_domain }}"
+  AWS_ACCESS_KEY_UD: "ASDF321ADF63"
```
Comment (pabrahamsson) [inline cert-manager-configs/.applier/group_vars/seed-hosts.yml:8]: Typo? Should be `AWS_ACCESS_KEY_ID` I assume.

**Review by pabrahamsson — CHANGES_REQUESTED** [review]

File: cert-manager-configs/.applier/group_vars/seed-hosts.yml (hunk 1 above)
Comment (oybed) [reply cert-manager-configs/.applier/group_vars/seed-hosts.yml]: yes, fixed.

**Review by pabrahamsson — COMMENTED** [review]
Looks good @oybed, should we make a mention somewhere that the `AWS_SECRET_ACCESS_KEY` needs to be base64 encoded? Alternatively we could use `stringData` in the secret.

**Review by pabrahamsson — APPROVED** [review]
Thank you @oybed 
/lgtm

**Pull request merged**